src/Microsoft.AspNet.Server.Testing/Deployers/ApplicationDeployer.cs

```diff
@@ -70,16 +70,13 @@ protected void DnuPublish(string publishRoot = null)
         {
             DeploymentParameters.PublishedApplicationRootPath = Path.Combine(publishRoot ?? Path.GetTempPath(), Guid.NewGuid().ToString());
 
-            var parameters =
-                string.Format(
-                    "publish {0} -o {1} --runtime {2} {3}",
-                    DeploymentParameters.ApplicationPath,
-                    DeploymentParameters.PublishedApplicationRootPath,
-                    DeploymentParameters.DnxRuntime,
-                    DeploymentParameters.PublishWithNoSource ? "--no-source" : string.Empty);
+            var noSource = DeploymentParameters.PublishWithNoSource ? "--no-source" : string.Empty;
+            var command = DeploymentParameters.Command ?? "web";
+            var parameters = $"publish {DeploymentParameters.ApplicationPath} -o {DeploymentParameters.PublishedApplicationRootPath}"
```

Comment:
[Nonblocking comments] Somehow I find it surprisingly difficult to read with string interpolation here. I think majorly is because the parameters names are way to long and the force wrap in github review view doesn't help either.